fom-hochschule-fuer-oekonomie-und-management.csl

```diff
@@ -0,0 +1,956 @@
+        <text macro="issued" prefix="(" suffix=")"/>
+      </if>
+      <else-if type="article-journal">
+        <choose>
+          <if variable="issue volume" match="any">
+            <text macro="issued" prefix="(" suffix=")"/>
+          </if>
+          <else>
+            <text macro="issued" prefix="(" suffix=")"/>
+          </else>
+        </choose>
+      </else-if>
+      <else-if type="webpage post-weblog" match="any">
+        <text macro="issued" prefix="(" suffix=")"/>
+      </else-if>
+      <else-if type="speech">
+        <group delimiter=", ">
+          <group delimiter=" ">
+            <choose>
+              <if variable="title" match="none"/>
+              <else>
+                <text variable="genre" text-case="capitalize-first"/>
+              </else>
+            </choose>
+            <text macro="event"/>
+          </group>
+          <text variable="event-place"/>
+          <text macro="issued"/>
+        </group>
+      </else-if>
+      <else-if type="article-newspaper">
+        <text macro="issued"/>
+      </else-if>
+      <else-if type="book bill chapter graphic legislation motion_picture paper-conference report song thesis" match="any">
+        <group delimiter=", ">
+          <choose>
+            <if type="thesis">
+              <text variable="genre" text-case="capitalize-first"/>
+            </if>
+          </choose>
+          <group delimiter=". ">
+            <text macro="originally-published"/>
+            <group delimiter=", ">
+              <text macro="reprint"/>
+              <text macro="publisher"/>
+            </group>
+          </group>
+          <text macro="issued"/>
+        </group>
+      </else-if>
+      <else>
+        <text macro="issued" prefix="(" suffix=")"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="graphic report thesis" match="any">
+        <text macro="archive"/>
+      </if>
+      <else-if type="webpage post-weblog" match="any">
+        <group delimiter=", ">
+          <text variable="URL"/>
+          <group delimiter=" " prefix="(" suffix=")">
+            <text term="accessed" text-case="capitalize-first"/>
+            <date variable="accessed" suffix=", MEZ">
+              <date-part name="year"/>
+              <date-part name="month" form="numeric-leading-zeros" prefix="-"/>
+              <date-part name="day" form="numeric-leading-zeros" prefix="-"/>
+            </date>
+          </group>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="case-locator-subsequent">
+    <choose>
+      <if type="legal_case">
+        <group delimiter=" ">
+          <text variable="volume"/>
+          <text variable="container-title"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="locator-note">
+    <choose>
+      <if type="article-journal book chapter webpage post-weblog " match="any">
```

Comment:
Dear @rmzelle 
Can you tell me why books don't print the pages in the notes here? It works fine for chapters. (Note the user will use Mendeley and Mendeley doesn't have the variable="locator" function like Zotero)

Reply:
@POBrien333, Mendeley's CSL mapping webpage seems to be offline, but my guess is that Mendeley, like Zotero, doesn't have the "page" variable mapped for the "book" item type. (Zotero only has "number-of-pages" for "book", see https://aurimasv.github.io/z2csl/typeMap.xml#map-book)

Reply:
Grand. That's what I had thought.

Reply:
@POBrien333, Mendeley's CSL mapping is back at https://service.elsevier.com/app/answers/detail/a_id/22175/supporthub/mendeley/ (for now at least)